src/include/txn.i

```diff
@@ -110,6 +110,10 @@ __wt_txn_oldest_id(WT_SESSION_IMPL *session)
 	txn_global = &S2C(session)->txn_global;
 	btree = S2BT_SAFE(session);
 
+	/* All updates to the lookaside table are immediately visible. */
+	if (btree != NULL && F_ISSET(btree, WT_BTREE_LOOKASIDE))
+		return (WT_TXN_ABORTED);
+
```

Comment:
This is causing assertion failures -- `session->dhandle` must be set to the LAS table in some situation where we are committing a transaction.  Rather than attempt to debug that, can I suggest that we leave this out for now and come back to it later?

Another way to get the same effect would be to arrange for all updates to the LAS to have txnid = WT_TXN_NONE: that wouldn't require any special visibility checks for LAS reads.